meson.build

```
@@ -505,15 +505,17 @@ if get_option('enable-test')
     configuration: nnstreamer_test_conf
   )
 
-  path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_plugin = join_paths(meson.build_root(), 'gst')
+  path_ext_plugin = join_paths(meson.build_root(), 'ext')
+
   path_nns_conf = join_paths(meson.build_root(), 'nnstreamer-test.ini')
   path_nns_plugin_prefix = join_paths(meson.build_root(), 'ext', 'nnstreamer')
```

Minor revision request.
```suggestion
  path_nns_plugin_prefix = join_paths(path_ext_plugin, 'nnstreamer')
```

---

Thanks! this is more neat. I updated the code